tests/console/yast2_lan.pm

```diff
@@ -78,8 +78,10 @@ sub run {
     type_string $domain;
     assert_screen 'test-yast2_lan-1';
 
+    my $timeout = 90;
+    $timeout *= 2 if check_var('ARCH', 'aarch64');
     send_key "alt-o";    # OK=>Save&Exit
-    wait_serial("yast2-lan-status-0", 90) || die "'yast2 lan' didn't finish";
+    wait_serial("yast2-lan-status-0", $timeout) || die "'yast2 lan' didn't finish";
```

Comment:
I believe it's fine to just bump the timeout to 180, as tests are unstable sometimes, and if it finishes earlier, we won't wait 3 minutes anyway.